fix: creates div to wrap formfield

gweiying opened the pull request:
This PR contains a small fix for the bug shown below where the error message runs onto the next form field, by creating a `<div></div>` tag to wrap the form fields. 

Before:
<img width="341" alt="Screenshot 2020-12-29 at 12 28 09 PM" src="https://user-images.githubusercontent.com/39231249/103280696-518b2c80-49d1-11eb-9c3a-fbd53d1e3926.png">

Fixed:
<img width="338" alt="Screenshot 2020-12-29 at 12 29 50 PM" src="https://user-images.githubusercontent.com/39231249/103280794-8e572380-49d1-11eb-8883-3bac783faece.png">

kwajiehao commented:
Previously we added extra line breaks in the `ComponentSettingsModal` when we faced this problem (#299), but I think this makes more sense given that the problem is appearing in `EditHomepage` and ostensibly other places as well. 

1. Can you help remove the line breaks from `ComponentSettingsModal`? Currently, the extra div and line break results in two effective line breaks

<img width="646" alt="Screenshot 2020-12-29 at 8 38 45 PM" src="https://user-images.githubusercontent.com/31984694/103284401-22e37500-4a16-11eb-9b60-fff4112ce1a1.png">

2. Do we need to apply the same fix to the other form fields (mainly `FormFieldHorizontal`), or is the current situation ok?  Personally, I think the current situation is fine.

_Current situation_
<img width="614" alt="Screenshot 2020-12-29 at 8 43 10 PM" src="https://user-images.githubusercontent.com/31984694/103284575-8c638380-4a16-11eb-9b9f-dd64fbb99ac2.png">

_With additional line break_
<img width="621" alt="Screenshot 2020-12-29 at 8 47 18 PM" src="https://user-images.githubusercontent.com/31984694/103284752-11e73380-4a17-11eb-9159-810371baf324.png">

gweiying commented:
> Previously we added extra line breaks in the `ComponentSettingsModal` when we faced this problem (#299), but I think this makes more sense given that the problem is appearing in `EditHomepage` and ostensibly other places as well.
> 
> Can you help remove the line breaks from `ComponentSettingsModal`? Currently, the extra div and line break results in two effective line breaks
> 
> <img alt="Screenshot 2020-12-29 at 8 38 45 PM" width="646" src="https://user-images.githubusercontent.com/31984694/103284401-22e37500-4a16-11eb-9b60-fff4112ce1a1.png">
 
Sounds good, addressed here b938f9c

kwajiehao reviewed (approved):
lgtm
